### PR TITLE
Feature/translation review and fixes

### PR DESCRIPTION
**Pull request by Xarlos89**
Made a few changes to Cake's translator cog. Check em. Test em, and approve it already.

### CODE REVIEW

File: src/zorak/cogs/general/general_trans.py [hunk 1]
```diff
@@ -1,74 +1,81 @@
+
 import discord.ext
 import googletrans
 from discord.ext import commands
 from googletrans import Translator
 
+
+def is_multi_lang(lang):
+    if isinstance(lang, list):
```
Comment (harrietobrien) [inline src/zorak/cogs/general/general_trans.py]: not wrong but cleaner:
`return isinstance(lang, list)`

File: src/zorak/cogs/general/general_trans.py [hunk 2]
```diff
@@ -1,74 +1,81 @@
...
+        return True
+    else:
+        return False
+
+
 class googletrans_func(commands.Cog):
```
Comment (harrietobrien) [inline src/zorak/cogs/general/general_trans.py]: I would just name it:
`class GoogleTranslate(commands.Cog)`

by PEP 8, classes should use CapsWord convention and not include underscores

File: src/zorak/cogs/general/general_trans.py (hunk 1 above)
Comment (Xarlos89) [reply src/zorak/cogs/general/general_trans.py]: 100% correct. This is way easier. And technically, using this we dont need the func, but the func increases readability so i'll leave it in.

File: src/zorak/cogs/general/general_trans.py (hunk 2 above)
Comment (Xarlos89) [reply src/zorak/cogs/general/general_trans.py]: Indeed, and changed!

**Pull request merged**